internal/handlers/pg/pgdb/collections.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 // validateCollectionNameRe validates collection names.
-var validateCollectionNameRe = regexp.MustCompile("^[a-zA-Z_-][a-zA-Z0-9_-]{0,119}$")
+var validateCollectionNameRe = regexp.MustCompile("^([/]|[a-zA-Z_-])([/]|[a-zA-Z0-9_-]){0,119}$")
```

Comment:
I think it could be simplified to add `/` into the groups themselves. For the first group that would be `[a-zA-Z_/-]`